keras2onnx/_builtin.py

```diff
@@ -1689,6 +1689,41 @@ def convert_tf_one_hot(scope, operator, container):
                               name=operator.full_name + '_one_hot', axis=axis)
 
 
+@converter_func(TYPES.Pow)
+def convert_tf_pow(scope, operator, container):
+    oopb = OnnxOperatorBuilder(container, scope)
+    node = operator.raw_operator
```

Comment:
should all of these be merged into onnx_ops?

Reply:
onnx_ops doesn't have type info inside.hhhhh

Reply:
Reverted